docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md

```diff
@@ -14,9 +14,9 @@ async createTestCafe([hostname], [port1], [port2], [sslOptions], [developmentMod
 
 Parameter                     | Type   | Description                                                                                                                                                                                                  | Default
 ----------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------
-`hostname`&#160;*(optional)*       | String | The hostname or IP you will use to address the TestCafe server. Must resolve to the current machine. To test on external devices, use the hostname that is visible in the network shared with these devices. | Hostname of the OS. If the hostname does not resolve to the current machine - its network IP address.
+`hostname`&#160;*(optional)*       | String | The hostname or IP you use to access the TestCafe server. Must resolve to the current machine. To test on external devices, use the hostname that is visible in the network shared with these devices. | Hostname of the OS. If the hostname does not resolve to the current machine - its network IP address.
```

Comment:
`The hostname or IP you use to access the TestCafe server. `
We use an another wording:
`The hostname of IP on which TestCafe is running`